### ISSUE TITLE
OCDS: Internal server error when null appears in parties array

### ISSUE THREAD

**Issue by duncandewhurst** [OpenDataServices/cove#1287]
Minimal example to reproduce issue - add `null` to the `parties` array:

```json
{
  "version": "1.1",
  "uri": "https://github.com/open-contracting/sample-data/raw/1.1/fictional-example/ocds-213czf-000-00001.json",
  "publishedDate": "2011-01-10T09:30:00Z",
  "publisher": {
    "name": "Open Data Services Co-operative Limited",
    "scheme": "GB-COH",
    "uid": "9506232.0",
    "uri": "http://data.companieshouse.gov.uk/doc/company/09506232"
  },
  "license": "http://opendatacommons.org/licenses/pddl/1.0/",
  "publicationPolicy": "https://github.com/open-contracting/sample-data/",
  "releases": [
    {
      "ocid": "ocds-213czf-000-00001",
      "id": "ocds-213czf-000-00001-01-planning",
      "language": "en",
      "date": "2009-03-15T14:45:00Z",
      "initiationType": "tender",
      "tag": [
        "planning"
      ],
      "parties": [
        {
          "id": "GB-LAC-E09000003"
        },
        null
      ]
    }
  ]
}
```

Can this error be handled and reported to the user?

Related to CRM issue [#6128](https://crm.open-contracting.org/issues/6128)

**Comment by duncandewhurst** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Investigate CoVE errors in Kingfisher Process Sentry](https://trello.com/c/gyTXFIUt/232-investigate-cove-errors-in-kingfisher-process-sentry)

**Comment by odscjames** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Internal server error when null appears in parties array](https://trello.com/c/Ys5zqcdl/89-internal-server-error-when-null-appears-in-parties-array)

**Comment by rhiaro** [comment]
This was fixed in lib-cove 0.18.0 and OCDS Cove with the new version has been deployed.
